app-modules/portal/src/Http/Controllers/KnowledgeManagementPortal/KnowledgeManagementPortalLogoutController.php

```diff
@@ -58,8 +61,14 @@ public function __invoke(Request $request): JsonResponse
 
         auth('contact')->logout();
 
+        if (! Session::has('guest_id') && ArticleWasHelpful::active()) {
+            $portalGuest = PortalGuest::create();
+            Session::put('guest_id', $portalGuest->id);
+        }
+
         return response()->json([
             'success' => true,
+            'guest_id' => Session::get('guest_id'),
```

Comment:
What is the purpose of passing the `guest_id` here? If you need to retrieve it somewhere later in the code you should be able to reference it in the session right?